docs/userguide/containers/dockervolumes.md

```diff
@@ -11,17 +11,18 @@ parent = "engine_learn"
 
 # Manage data in containers
 
-So far we've been introduced to some [basic Docker concepts](../containers/usingdocker.md),
-seen how to work with [Docker images](../containers/dockerimages.md) as well as learned about
-[networking and links between containers](../networking/default_network/dockerlinks.md). In this section we're
-going to discuss how you can manage data inside and between your Docker
-containers.
+So far you've been introduced to some [basic Docker
+concepts](../containers/usingdocker.md), seen how to work with [Docker
+images](../containers/dockerimages.md) as well as learned about [networking and
+links between containers](../networking/default_network/dockerlinks.md). In this
```

Comment:
"In this section you're going to learn how you can manage data inside and between your Docker containers"